src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -121,7 +121,7 @@ const CoverImageEditModal = ({
           Success({ msg: "Cover image updated." });
         } else {
           Notification.Error({
-            msg: "Something went wrong!",
+            msg: "Oops! Facility cover image is too large. Please use a smaller size image.",
```

Comment:
@Aryanshu919 

There could be other possibilities where this API can error out i.e., eg: when an SVG file is uploaded, the API errors out, but still file too large message would be shown.

For the issue #7661, make a backend fix where the respective error message is sent based on the actual issue and then show the error message send from the backend here